com.civicrm.hrjobroles/CRM/Hrjobroles/Page/JobRoles.php

```diff
@@ -19,6 +19,5 @@ private static function registerScripts() {
 
     CRM_Core_Resources::singleton()->addScriptFile('com.civicrm.hrjobroles', CRM_Core_Config::singleton()->debug ? 'js/src/job-roles.js' : 'js/dist/job-roles.min.js', 1010);
     CRM_Core_Resources::singleton()->addStyleFile('com.civicrm.hrjobroles', 'css/hrjobroles.css');
-    CRM_Core_Resources::singleton()->addStyleFile('com.civicrm.hrjobroles', 'angular-xeditable/css/xeditable.css');
```

Comment:
We don't need it anymore, the style for xeditable is not included directly in Shoreditch